lib/common/events.ts

```diff
@@ -396,9 +396,16 @@ export function patchEventTarget(
         taskData.isExisting = isExisting;
 
         const data = useGlobalCallback ? OPTIMIZED_ZONE_EVENT_TASK_DATA : null;
+
+        // keep taskData into data to allow onScheduleEventTask to acess the task information
+        (data as any).taskData = taskData;
```

Comment:
if the `useGlobalCallback` is false, data would be `null` and this line would fail.

Reply:
@jahtalab, yes, you are right, thank you.